sdk/android/src/java/org/webrtc/HardwareVideoEncoder.java

```diff
@@ -284,6 +286,10 @@ private VideoCodecStatus initEncodeInternal() {
       stride = getStride(inputFormat, width);
       sliceHeight = getSliceHeight(inputFormat, height);
 
+      // RingRTC change (or ... retention?) to keep support for SDK >= 19.
+      if (Build.VERSION.SDK_INT < 21) {
+        outputBuffers = codec.getOutputBuffers();
+      }
```

Comment:
Is there a reason to conditionalize this rather than "just" reverting the upstream change? Seems like that'd be cleaner to track, and would help catch places where upstream starts depending on the new API.